WordPress/Credentials/gencredentials.rb

```diff
@@ -242,4 +254,4 @@ def print_class(client, secret, sentry, appcenter, giphy, google_client, google_
   end
 end
 
-print_class(client, secret, sentry, appcenter, giphy, google_client, google_scheme, google_login_server, debugging_key, zendesk_app_id, zendesk_url, zendesk_client_id)
+print_class(client, secret, sentry, appcenter, giphy, google_client, google_scheme, google_login_server, debugging_key, zendesk_app_id, zendesk_url, zendesk_client_id, tenor_key)
```

Comment:
I think it would be helpful to add a check for `tenor_key` and print an error message like we do for the other credentials. For example:

```
  if google_client.nil?
    $stderr.puts "warning: Google Login Client ID not found"
  end
```

Reply:
Thanks for spotting this @ScoutHarris. I have added the check for Tenor part